curvefs/src/client/metric/client_metric.h

```diff
@@ -275,18 +276,32 @@ struct S3Metric {
           writeSize(prefix, fsName + "_adaptor_write_size", 0) {}
 };
 
+template <typename Tp>
+uint64_t atomValue2Bytes(void* atomValue) {
+    std::atomic<Tp>* bytes = reinterpret_cast<std::atomic<Tp>*>(atomValue);
+    return bytes->load();
+}
+
 struct DiskCacheMetric {
     static const std::string prefix;
 
     std::string fsName;
     InterfaceMetric writeS3;
-    bvar::Status<uint64_t> diskUsedBytes;
+    bvar::PassiveStatus<uint64_t> usedBytes_;
+    bvar::PassiveStatus<uint64_t> totalBytes_;
+    InterfaceMetric trim_;
 
-    explicit DiskCacheMetric(const std::string &name = "")
+    explicit DiskCacheMetric(const std::string& name = "",
+                             std::atomic<int64_t>* usedBytes = nullptr,
```

Comment:
uint64_t?

Reply:
> uint64_t?

The actual type is int64_t

https://github.com/Cyber-SiKu/curve/blob/385018b37aa78f3cd70a5303defbb5701ccd0ed1/curvefs/src/client/s3/disk_cache_manager.h#L182-L184